doc/v2/howto/capi/index_en.rst

```diff
@@ -1,6 +1,23 @@
 C-API Prediction Library
 ========================
 
+After we trained a neural network model, then we use it to make predictions. Prediction is a process of preparing input data and processing the model to get the result.
+
+Compared with model training, prediction has the following features:
+
+#. Prediction does not require backpropagation and parameter updates during training.
```

Comment:
Inference does not require backpropagation and parameter updates, as required during training.